### DISCUSSION TITLE
使用Docker容器创建sakura1(frpc)，如何修改其加载的配置？

### DISCUSSION THREAD

**Discussion by Synchronized1** [natfrp/wiki#295] [使用问题] [answered]
启动容器所用代码：
docker run -d --restart=always --name=sakura1 registry.cn-hongkong.aliyuncs.com/natfrp/frpc -f xxxx

但是docker logs 查看会报eof
![image](https://user-images.githubusercontent.com/58451554/227514081-ad5e10a3-444c-4b7d-87d1-8ba1db0e8c3e.png)

之前用自己的vps会遇到这个问题，需要在frpc.ini中配置 
![image](https://user-images.githubusercontent.com/58451554/227514352-7efe712f-a058-43c7-b01e-43ec4743bd01.png)

请问使用容器创建的sakura，如何修改frpc配置？？？或者说容器内的配置文件的路径是？

**Comment by fakeboboliu** [comment, marked as answer]
您好，
通常这是您的网络管理员（公司或学校）正在屏蔽 frp 流量。
启用 tls 为绕过您的网管安全措施的行为，我们推荐您先了解相关的限制政策，避免造成内网安全隐患，危及人身安全。
如果您确认要这么做的话，请在指令最后添加 `--natfrp_tls` 参数。

**Reply by Synchronized1** [reply]
感谢大佬指点😁,加上该参数后就成功了!
![image](https://user-images.githubusercontent.com/58451554/227681699-13d239e3-864f-4bbc-a01d-373d28e7670b.png)
然后我找到了官方手册(已收藏):
https://doc.natfrp.com/frpc/manual.html#advanced-switches
